Sources/BugsnagPerformance/Private/Tracer.mm

```diff
@@ -314,29 +321,41 @@
 
 BugsnagPerformanceSpan *
 Tracer::startAppStartSpan(NSString *name,
-                        SpanOptions options) noexcept {
-    return startSpan(name, options, BSGTriStateUnset);
+                          SpanOptions options,
+                          NSArray<BugsnagPerformanceSpanCondition *> *assignedConditions) noexcept {
+    return startSpan(name, options, BSGTriStateUnset, assignedConditions);
 }
 
 BugsnagPerformanceSpan *
 Tracer::startCustomSpan(NSString *name,
                         SpanOptions options) noexcept {
-    return startSpan(name, options, BSGTriStateYes);
+    return startSpan(name, options, BSGTriStateYes, @[]);
 }
 
 BugsnagPerformanceSpan *
 Tracer::startViewLoadSpan(BugsnagPerformanceViewType viewType,
                           NSString *className,
                           SpanOptions options) noexcept {
     NSString *type = getBugsnagPerformanceViewTypeName(viewType);
+    NSMutableArray<BugsnagPerformanceSpanCondition *> *assignedConditions = [NSMutableArray array];
```

Comment:
Given that these arrays have zero-or-one items in them, does it possibly make sense to either pre-size them? Or possibly do:

```
NSMutableArray<BugsnagPerformanceSpanCondition *> *blockingConditions = @[];
....
if (appStartupCondition) {
    blockingConditions = @[appStartupCondition];
}
```